cmd/crane/cmd/root.go

```diff
@@ -137,7 +142,8 @@ func New(use, short string, options []crane.Option) *cobra.Command {
 	root.PersistentFlags().BoolVar(&insecure, "insecure", false, "Allow image references to be fetched without TLS")
 	root.PersistentFlags().BoolVar(&ndlayers, "allow-nondistributable-artifacts", false, "Allow pushing non-distributable (foreign) layers")
 	root.PersistentFlags().Var(platform, "platform", "Specifies the platform in the form os/arch[/variant][:osversion] (e.g. linux/amd64).")
-
+	root.PersistentFlags().StringVar(&local, "local", "", "Use a local oci-layout as remote registry")
```

Comment:
This feels like a big change, since it affects all commands. Would it be worth having some e2e tests that set up a local layout and `crane pull --local=<that>`, `crane mutate`, etc.?